vue-app/src/api/projects.ts

```diff
@@ -33,13 +33,16 @@ export interface Project {
   extra?: any // Registry-specific data
 }
 
+//TODO: update anywhere this is called to take factory address as a parameter
+//NOTE: why isn't this included in the vuex state schema?
```

Comment:
FYI @daodesigner I believe it is: https://github.com/ethereum/clrfund/blob/develop/vue-app/src/store/index.ts#L100